server/models/Ticket.js

```diff
@@ -1,30 +1,23 @@
-const mongoose = require('mongoose');
+const { Schema } = require("mongoose");
 
-const { Schema } = mongoose;
 
-const ticketSchema = new  Schema({
-    purchaseDate:{
-        type : Date,
-        default : Date.now
-    },
-    price: {
-        type: Number,
-        required: true,
-        min: 0.99
-      },
+const ticketSchema = new Schema({
+  purchaseDate: {
+    type: Date,
+    default: Date.now,
```

Comment:
1. `default: Date.now` should be `default: Date.now()
2. If possible a getter with formatting date would be nice
Possible solution:
- use date-format module
- `const format = require("date-format");`
-  `get: (date) => format.asString("yyyy/MM/dd hh:mm:ss", date),` (add this as a third item inside purchaseDate{}

Reply:
okay sure I will